src/components/Stats.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
+const calculatePercentage = (cases, total) => {
+  return ((cases / total) * 100).toFixed(2).toLocaleString('en');
```

Comment:
A small comment which doesn't matter for functionality wise. You can remove the brackets if you want as it's a single line function.